### ISSUE TITLE
[HW] slachiewicz

### ISSUE THREAD

**Issue by slachiewicz** [datastaxdevs/k8ssandra-workshop#3]
**Name:** Sylwester Lachiewicz
**Linkedin Profile:** https://www.linkedin.com/in/slachiewicz/

Attach the homework screenshots below:
-----------------------------------------

![image](https://user-images.githubusercontent.com/6705942/109869393-14f5de80-7c69-11eb-8fba-0884735e7e35.png)

Too fast - scale to 3 instead of 2 on local laptop ;-)

**Comment by HadesArchitect** [comment]
Hi @slachiewicz, glad you made it working! I'd like to have a couple of more things, e.g.: 

* Can you access grafana?  
* Can you connect to Cassandra using cqlsh?

**Comment by slachiewicz** [comment]
Yes, everything was warming fine until I decided to scale cluster. Sorry no screenshots

**Comment by HadesArchitect** [comment]
Ok I see! I think it'll be enough. Thank you!

**Comment by HadesArchitect** [comment]
Here we go: https://api.badgr.io/public/assertions/PQUM9RpqSFinge_M1st2JQ?identity__email=slachiewicz%40gmail.com
